src/models/Organization.ts

```diff
@@ -0,0 +1,42 @@
+import { BelongsToManyAddAssociationMixin, DataTypes, Model } from "sequelize";
+import sequelize from "../database/db";
+import { User } from "./User";
+
+export class Organization extends Model {
+  public ipaCode!: string; // PK
+  public name!: string;
+  public pec!: string;
+  public description!: string;
+  public readonly createdAt!: Date;
+  public readonly deletedAt!: Date;
+  public readonly updatedAt!: Date;
+
+  public addUser!: BelongsToManyAddAssociationMixin<User, string>;
+
+  public readonly users?: ReadonlyArray<User>;
+}
+
+Organization.init(
+  {
+    ipaCode: {
```

Comment:
shouldn't these be equal to the name of the fields in the table?

Reply:
Yes, in fact I used the same names both in this file and in the related migration file.